canonical-web-and-design/vanilla-framework
# Chips improvements

## Done

Based on #3175; improve styling of chips. Includes code for the popover menu. This is just to ensure the chips fit as expected in the bigger picture. This will be deleted before merging.

## QA

- Pull code
- Run `./run`
- Open docs/examples/patterns/chip/multiple
To evaluate in situ, add the following [scss](https://pastebin.canonical.com/p/Y7HqmMBFMD/) and replace the markup of any chips example with [this](https://pastebin.canonical.com/p/KZtFNSd8zW/)

- Verify it looks like this:

![image](https://user-images.githubusercontent.com/2741678/88532710-f3f74c00-cffc-11ea-97fc-ef745bdf167a.png)

Fixes #3179 
Fixes #3192
